doc/source/conf.py

```diff
@@ -334,9 +332,74 @@ def setup(app):
 
 #  --Options for sphinx extensions -----------------------------------------------
 
+# connect docs in other projects
 intersphinx_mapping = {
+    "pyproj": (
+        "https://pyproj4.github.io/pyproj/stable/",
+        "https://pyproj4.github.io/pyproj/stable/objects.inv",
+    ),
     "pandas": (
         "https://pandas.pydata.org/pandas-docs/stable/",
         "https://pandas.pydata.org/pandas-docs/stable/objects.inv",
     ),
-}
\ No newline at end of file
+    "shapely": (
+        "https://shapely.readthedocs.io/en/stable/",
+        "https://shapely.readthedocs.io/en/stable/objects.inv",
+    ),
+    "fiona": (
+        "https://fiona.readthedocs.io/en/latest/",
+        "https://fiona.readthedocs.io/en/latest/objects.inv",
```

Comment:
Can we link stable instead of latest? The same applies to all below linking latest.

Reply:
I have found the stable versions of all of them except [pygeos](https://pygeos.readthedocs.io/en/latest/) and [cartophy](https://scitools.org.uk/cartopy/docs/latest/). They have documentations for different versions, where the `latest` points to the most recent version and there is no `stable`. I can fix them to specific version or leave them with `latest`, which would be better?

Reply:
If they don't have stable, latest is the best option I guess.

Reply:
I think so too. I have made the [changes](https://github.com/geopandas/geopandas/pull/1931/commits/553b25a92de3cf3287fdfe186b82fb58f8c7ac0a).